stories/addon/cc-jenkins-info.stories.js

```diff
@@ -0,0 +1,72 @@
+import '../../src/addon/cc-jenkins-info.js';
+import { makeStory, storyWait } from '../lib/make-story.js';
+import { enhanceStoriesNames } from '../lib/story-names.js';
+
+export default {
+  title: '🛠 Addon/<cc-jenkins-info>',
+  component: 'cc-jenkins-info',
+};
+
+const conf = {
+  component: 'cc-jenkins-info',
+  css: `
+    cc-jenkins-info {
+      margin-bottom: 1rem;
+    }
+  `,
+};
+
+const jenkinsLink = 'https://my-jenkins.example.com';
+const jenkinsManageLink = 'https://my-jenkins.example.com/manage';
+const differentVersions = { current: 'v2.1.3', available: 'v2.1.4' };
+const sameVersions = { current: 'v2.1.3', available: 'v2.1.3' };
+
+export const defaultStory = makeStory(conf, {
+  items: [{ jenkinsLink, jenkinsManageLink, versions: sameVersions }],
+});
+
+export const skeleton = makeStory(conf, {
+  items: [{}],
+});
+
+export const dataLoaded = makeStory(conf, {
```

Comment:
Could be renamed into `dataLoadedWithSameVersions`.

This will be displayed as **Data loaded (same versions)** in Storybook.